scaleway/compute_instance_helpers.go

```diff
@@ -23,3 +23,18 @@ func getInstanceAPIWithZoneAndID(m interface{}, id string) (*instance.API, utils
 	zone, ID, err := parseZonedID(id)
 	return instanceApi, zone, ID, err
 }
+
+func flattenRootVolume(v interface{}) []map[string]interface{} {
+	flattenVolume := []map[string]interface{}{{}}
+
+	vs, ok := v.([]map[string]interface{})
+	if ok {
+		flattenVolume = vs
+	}
+
+	if _, exist := flattenVolume[0]["delete_on_termination"]; !exist {
```

Comment:
can` flattenVolume` be empty? This might segfault.

Reply:
It's never empty (flattenVolume[0] is set line 28)

Reply:
My bad, `{{}}` it easily missed when reading the code? :p

Reply:
Is it better this way?

Reply:
Yeps!